backend/src/test/java/com/javahelp/backend/endpoint/user/ReadHandlerTest.java

```diff
@@ -0,0 +1,136 @@
+package com.javahelp.backend.endpoint.user;
+
+import static org.junit.Assert.assertEquals;
+import static org.junit.Assume.assumeTrue;
+import static io.restassured.RestAssured.given;
+
+import com.javahelp.backend.data.ITokenStore;
+import com.javahelp.backend.data.IUserStore;
+import com.javahelp.model.token.Token;
+import com.javahelp.model.user.ClientUserInfo;
+import com.javahelp.model.user.SHAPasswordHasher;
+import com.javahelp.model.user.User;
+import com.javahelp.model.user.UserPassword;
+import com.javahelp.model.util.json.UserConverter;
+
+import org.junit.Test;
+
+import java.time.Duration;
+
+import io.restassured.http.Header;
+import io.restassured.response.Response;
+
+/**
+ * Tests for {@link ReadHandler}
+ */
+public class ReadHandlerTest {
+
+    IUserStore store = IUserStore.getDefaultImplementation();
+    ITokenStore tokenStore = ITokenStore.getDefaultImplementation();
+
+    private static final String USER_ENDPOINT = "https://gwkvm1k2j5.execute-api.us-east-1.amazonaws.com/users/";
+
+    ClientUserInfo info = new ClientUserInfo("i want to sleep",
```

Comment:
lol you really put this in context. Hopefully you will get some sleep soon this holiday :D